tutorials/dataframe/df034_SaveGraph.py

```diff
@@ -0,0 +1,44 @@
+'''
+Use SaveGraph to inspect the sequence of RDataFrame actions.
+
+The tutorial shows how to use SaveGraph action.
+'''
```

Comment:
Tutorials need a header with a bit more structure, see e.g. df001_introduction.py for an example